src/file-selector.spec.ts

```diff
@@ -1,6 +1,6 @@
-import {FileWithPath} from './file';
-import {fromEvent} from './file-selector';
-
+import { expect, it } from '@jest/globals';
```

Comment:
Jest doesn't expose globals in ESM mode, explicit imports are [needed](https://jestjs.io/docs/ecmascript-modules#differences-between-esm-and-commonjs) instead.